suite-native/formatters/src/components/FeeFormatter.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { fromWei } from 'web3-utils';
+
+import { Text } from '@suite-native/atoms';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { getFeeRate, getFeeUnits } from '@suite-common/wallet-utils';
+import { networks } from '@suite-common/wallet-config';
+
+type FeeFormatterProps = {
+    transaction: WalletAccountTransaction;
+};
+
+export const FeeFormatter = ({ transaction }: FeeFormatterProps) => {
+    const { networkType } = networks[transaction.symbol];
+
+    const formattedValue =
```

Comment:
We should `useMemo` this or whole component, these BigNumbers are quite slow on mobile.